### PR TITLE
(@wdio/sauce-service): bump saucelabs package

### PR DESCRIPTION
**Pull request by erwinheitzman**
## Proposed changes

Bump the saucelabs package version to pull the latest security updates.

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

### CODE REVIEW

File: packages/wdio-sauce-service/package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "@wdio/types": "8.16.3",
     "@wdio/utils": "8.16.3",
     "ip": "^1.1.8",
-    "saucelabs": "7.2.2",
+    "saucelabs": "7.3.1",
```
Comment (christian-bromann) [inline packages/wdio-sauce-service/package.json:37]:
```suggestion
    "saucelabs": "^7.3.1",
```

**Review by christian-bromann — APPROVED** [review]
👍

File: packages/wdio-sauce-service/package.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
     "@wdio/types": "8.16.3",
     "@wdio/utils": "8.16.3",
     "ip": "^1.1.8",
-    "saucelabs": "7.2.2",
+    "saucelabs": "^7.3.1",
```
Comment (redbugz) [inline packages/wdio-sauce-service/package.json:37]: This appears to have re-introduced the ESM loading problem
https://github.com/saucelabs/node-saucelabs/commit/a7814104cc324035901b9ade7d89c34182783175#commitcomment-127284025

Comment (christian-bromann) [reply packages/wdio-sauce-service/package.json:37]: Thanks for letting us know, I will revert.

**Pull request merged**